app/controllers/api/v1/reservations_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api::V1::ReservationsController < ApplicationController
+  before_action :set_reservation, only: %i[show destroy]
+
+  # GET /reservations
+  def index
+    @reservations = Reservation.where(user_id: @user.id).includes(:city, car: [:description])
```

Comment:
- [ ] Instead of using `where` here, I suggest that you simply select the user's reservations: `@user.reservations`.